chowpronto_frontend/src/hooks/useMenuItems.js

```diff
@@ -1,35 +1,6 @@
 function useMenuItems(location) {
-  // takes a location argument (TBC) & returns a function getItems
-  return [
-    new MenuItem("Burger", "Delicious Burger on Bun", 750, "Bobs Burger Shack"),
-    new MenuItem(
-      "Pizza",
-      "Fresh tomato & Buffalo Mozzeralla on a Hand-Stretched base",
-      1150,
-      "Italia Fresca"
-    ),
-    new MenuItem("Burger", "Delicious Burger on Bun", 750, "Bobs Burger Shack"),
-    new MenuItem(
-      "Pizza",
-      "Fresh tomato & Buffalo Mozzeralla on a Hand-Stretched base",
-      1150,
-      "Italia Fresca"
-    ),
-    new MenuItem("Burger", "Delicious Burger on Bun", 750, "Bobs Burger Shack"),
-    new MenuItem(
-      "Pizza",
-      "Fresh tomato & Buffalo Mozzeralla on a Hand-Stretched base",
-      1150,
-      "Italia Fresca"
-    ),
-    new MenuItem("Burger", "Delicious Burger on Bun", 750, "Bobs Burger Shack"),
-    new MenuItem(
-      "Pizza",
-      "Fresh tomato & Buffalo Mozzeralla on a Hand-Stretched base",
-      1150,
-      "Italia Fresca"
-    ),
-  ];
+  // takes an object with filters as key value pairs
+  let filteredFoods = dummyItems.
```

Comment:
Syntax Error `dummyItems.` should be `dummyItems;` (colon). Fixed in 39a68d81dfafc7745547be5fb35eb09c4a9aa2fb